src/Service/ServiceUtils.cpp

```diff
@@ -218,6 +218,26 @@ std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() noexcept {
   return TotalCpuTime{jiffies, cpus};
 }
 
+static ErrorMessageOr<fs::path> FindSymbolsFilePathInStructuredDebugStore(
+    const std::filesystem::path& structured_debug_store, std::string_view build_id) {
+  auto path_in_structured_debug_store = structured_debug_store / ".build-id" /
```

Comment:
CHECK that build_id is at least of size 3 as a precondition? (Assuming this function should only be called for well-formed build ids, o/w return an error.)

Reply:
Yes. Good point. Done. Thanks!